### ISSUE TITLE
how to slove this error?

### ISSUE THREAD

**Issue by baoyuchen** [YoojLee/Uniformaly#3]
I wrote two sh files based on your run_defect.sh file from the author. One is the file for training
![6afa15017dd4226a96e27c0355eb4e4](https://github.com/YoojLee/Uniformaly/assets/43003320/a9ba8270-de44-450e-9e2b-9504d9698ec0)
After the training was completed, I verified the images with a validation file. Here I have validated only the bottle class from mvtec data. sh file is shown below.
![demo2](https://github.com/YoojLee/Uniformaly/assets/43003320/aac57485-24ba-4fd0-a946-d81c4e69623f)
Then I run the sh file to get the following.
![demo3](https://github.com/YoojLee/Uniformaly/assets/43003320/cdd0c1f4-80ba-4f10-ac1f-e8333f3d437a)
But my thought was that if I wanted to save the picture results of my reasoning.
Let's say I want to reason about just one image, then how should I go about saving the result? Can I do the reasoning with the help of eval.py? Or do I need to write my own script to reason about a single image?
![demo4](https://github.com/YoojLee/Uniformaly/assets/43003320/e9aef8f1-fe09-42e7-8cd2-5be68e35d4ad)
But when I run it again it produces an error
In a nutshell, my aim is that I want to use a trained model to achieve inference on images and to save the inferred images, how can this be achieved?

**Comment by baoyuchen** [comment]
![Uploading demo5.png…]()
this is error

**Comment by YoojLee** [comment]
Hi, 

It seems that the last image you uploaded is not available. Would you please upload it again? 

Thanks!
